frontendservice/src/views/CreateShop.vue

```diff
@@ -0,0 +1,166 @@
+/* Copyright 2020 Google LLC Licensed under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance with the License. You
+may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. */
+
+<template>
+  <div>
+    <shopOwnerNavbar />
+    <form @submit.prevent="createShop()">
+      <h1 class="heading">Add a Shop!</h1>
+      <div class="field">
+        <label class="label">Shop Name</label>
+        <div class="control">
+          <input class="input" type="text" placeholder="Type Shop Name" id="shopName" required />
+        </div>
+      </div>
+      <div class="field">
+        <label class="label">Address</label>
+        <div class="control">
+          <input class="input" type="text" placeholder="Type Address" id="address" required />
+        </div>
+      </div>
+      <div>
+        <label class="label">Phone Number</label>
+        <vue-tel-input v-model="phone" @onInput="onInput" id="phoneNumber"></vue-tel-input>
+      </div>
+      <div class="field">
+        <label class="label">Shop Type</label>
+        <div class="control">
+          <div class="select">
+            <select id="shopType" required>
+              <option>Stationary</option>
+              <option>Food</option>
+              <option>Furniture</option>
+              <option>Electronics</option>
+              <option>Jewellery</option>
+              <option>Agriculture</option>
+            </select>
+          </div>
+        </div>
+      </div>
+      <div class="field">
+        <div class="control">
+          <label class="checkbox">
+            <input type="checkbox" id="conditions-check" />
+            I agree to the
+            <a href="#">terms and conditions</a>
```

Comment:
Just asking, are we planning to add a page to view these "terms and conditions" in our MVP? If not, shouldn't we remove this for now and add in "Future scope" if required? (I'm not aware if you have already discussed this with the hosts. If so, feel free to resolve the comment. If not, then maybe discuss it with them now.)

Reply:
I just confirmed it with hosts now, so they said that we can have it in MVP as discussed in one of our team meetings. I'll later add a page with 1/2 points for terms and conditions.